xtask/tests/client.Dockerfile

```
@@ -1,43 +1,33 @@
-    && ARCH=$(uname -m) && if [ "$ARCH" = "aarch64" ]; then ARCH="aarch64"; else ARCH="x86_64"; fi \
-    && wget https://github.com/Kitware/CMake/releases/download/v3.30.0/cmake-3.30.0-linux-${ARCH}.tar.gz \
-    && tar -xzf cmake-3.30.0-linux-${ARCH}.tar.gz --strip-components=1 -C /usr/local \
-    && rm cmake-3.30.0-linux-${ARCH}.tar.gz \
-    && wget https://github.com/rui314/mold/releases/download/v2.35.1/mold-2.35.1-${ARCH}-linux.tar.gz \
-    && tar -xzf mold-2.35.1-${ARCH}-linux.tar.gz -C /usr/local --strip-components=1 \
-    && rm mold-2.35.1-${ARCH}-linux.tar.gz \
-    && rm -rf /var/lib/apt/lists/*
-
+RUN apt-get update && apt-get install -y wget iproute2 build-essential clang pkg-config git \
+<<<<<<< HEAD
+    && wget https://github.com/Kitware/CMake/releases/download/v3.30.0/cmake-3.30.0-linux-aarch64.tar.gz \
```

Hadolint (2.14.0)

[error] unexpected '&'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)